app/assets/stylesheets/components.scss

```diff
@@ -333,3 +333,27 @@
     color: $white;
   }
 }
+
+.component-flex {
+  align-items: center;
+  display: flex;
+  justify-content: center;
```

Comment:
it is not good to add `flex-wrap: wrap;` property here? WDYT @bhutani2002 @vedant-jain03

Reply:
I think it is good for responsiveness of child elements greater then 1. It should be added. Also @bhutani2002 it is very hard to check the results in video it will be really great if you provide after-before photo.

Reply:
Also, mark `resolve conversation` for outdated code climate 😅

Reply:
Hey, @vedant-jain03 also I have updated pull request description and have linked the issue. In the issue I have attached all the before images, and since there very too many, therefore for the solution I made a video for that. In the before issues that I have stated, I used before and after pics.

Thank You